azure-pipelines.yml

```diff
@@ -1,5 +1,5 @@
 pool:
-  vmImage: 'Ubuntu 16.04'
+  vmImage: 'ubuntu-latest'
```

Comment:
Did you read that ? https://teams.microsoft.com/l/message/19:5ef20d45af8646cab8652b7e3d916b1d@thread.skype/1635520316621?tenantId=09639da7-e072-4542-917d-cad1a646ab6b&groupId=34f589c9-4e03-4759-8b8f-94414bb46ae7&parentMessageId=1635520316621&teamName=Development%20Team&channelName=General&createdTime=1635520316621 Maybe it's worth to use explicitly `ubuntu-20` rather than `latest` tag ? 🤔